### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_AzureDatamigrationservice_ScreenReader:NVDA is announcing table cell element key1 as clickable whereas it is not clickable.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15863]
### Actual:
NVDA is announcing table cell element key1 as clickable whereas it is not clickable.
**This issue is not observed with JAWS.**
 
### Expected:
NVDA should not announce the cell element as clickable.

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.3.1;#Element:Table;#AINotInScope;#NVDA;#JAWS;#STP
### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10
Screen Reader:NVDA,JAWS
### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Migrate to azure sql and press enter.
5. Add azure account and click on next.
6. Check radio button for SQL server on azure virtual machine and fill all the required fields,tab till next button and press enter.
7. Check radio button for Online migration and tab till next button and press enter.
8. Check radio button "My database backups are on a  network share" and fill all the required details and tab till next button and press enter.
9. Start screen reader and tab till table cell element key1.

### User Impact:
Screen Reader users will get incorrect information about table cell elements

["Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Comment by aasimkhan30** [comment]
The buttons in the authentication table need to have descriptive aria- labels.

**Comment by aasimkhan30** [comment]
<img width="935" alt="image" src="https://user-images.githubusercontent.com/6816294/123712402-1ace8b80-d827-11eb-83d7-bba3f362ef5f.png">

**Comment by aasimkhan30** [comment]
Tried with nvda screen reader and I cannot repro this bug.

**Comment by prabhudevu** [comment]
I have verified the issue in the below build

Version: 1.43.0-insider (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/afafee844ce2b07eb55bdb6f40ee1ce88c0bbf4e
Date: 2023-03-29T05:28:54.263Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

Issue is fixed.

**GithubTags:**#Closed;#Regressed:03-29-23;

![15863_closed](https://user-images.githubusercontent.com/98821792/228515650-d66b30c9-2ac9-4fe3-ba31-61793b343754.png)
